backend/resources/n_shot_examples_simple_linear.py

```diff
@@ -20,9 +20,141 @@
+            <tr><td>Idle</td><td>Idle</td><td>selectProgram</td><td>n>=1</td><td>r=n;</td></tr>
+            <tr><td>Idle</td><td>Idle</td><td>toggleDryingTime</td><td>NONE</td><td>if(dT==20){dT=40;}else{dT=20;}</td></tr>
+            <tr><td>Idle</td><td>Intake</td><td>start</td><td>door.isClosed()</td><td>door.lock();c=1;</td></tr>
+            <tr><td>Intake</td><td>Washing</td><td>NONE</td><td>NONE</td><td>NONE</td></tr>
+            <tr><td>Washing</td><td>Drain</td><td>after10min</td><td>NONE</td><td>NONE</td></tr>
+            <tr><td>Drain</td><td>Complete</td><td>NONE</td><td>c>=r</td><td>NONE</td></tr>
+            <tr><td>Drain</td><td>Intake</td><td>NONE</td><td>c<r</td><td>c++;</td></tr>
+            <tr><td>Drying</td><td>Suspended</td><td>close</td><td>NONE</td><td>NONE</td></tr>
+            <tr><td>Drying</td><td>Idle</td><td>afterDTmin</td><td>NONE</td><td>NONE</td></tr>
+            <tr><td>Suspended</td><td>Drying</td><td>open</td><td>NONE</td><td>door.unlock();</td></tr>
+            <tr><td>Idle</td><td>Drying</td><td>after5min</td><td>NONE</td><td>NONE</td></tr>
+            <tr><td>Drying</td><td>Drying</td><td>extendDryingTime</td><td>NONE</td><td>dT=40;</td></tr>
+            <tr><td>Cleaning.H</td><td>Intake</td><td>start</td><td>door.isClosed()</td><td>door.lock();c=1;</td></tr>
+            </table>""",
+            "hierarchical_state_table":
+            """<table border="1">
+            <tr><th>Superstate</th><th>Substate</th></tr>
+            <tr><td>DishwasherOn</td><td>Door</td></tr>
+            <tr><td>DishwasherOn</td><td>Washer</td></tr>
+            <tr><td>Door</td><td>Open</td></tr>
+            <tr><td>Door</td><td>Closed</td></tr>
+            <tr><td>Door</td><td>Locked</td></tr>
+            <tr><td>Washer</td><td>Idle</td></tr>
+            <tr><td>Washer</td><td>Cleaning</td></tr>
+            <tr><td>Washer</td><td>Drying</td></tr>
+            <tr><td>Washer</td><td>Suspended</td></tr>
+            <tr><td>Cleaning</td><td>Intake</td></tr>
+            <tr><td>Cleaning</td><td>Washing</td></tr>
+            <tr><td>Cleaning</td><td>Drain</td></tr>
+            </table>""",
+            "parallel_states_table": 
+            """<table border="1">
+            <tr><th>Parallel State</th><th>Parallel Region</th><th>Substate</th></tr>
+            <tr><td>DishwasherOn</td><td>Door</td><td>Open</td></tr>
+            <tr><td>DishwasherOn</td><td>Door</td><td>Closed</td></tr>
+            <tr><td>DishwasherOn</td><td>Door</td><td>Locked</td></tr>
+            <tr><td>DishwasherOn</td><td>Washer</td><td>Idle</td></tr>
+            <tr><td>DishwasherOn</td><td>Washer</td><td>Drying</td></tr>
+            <tr><td>DishwasherOn</td><td>Washer</td><td>Suspended</td></tr>
+            <tr><td>Cleaning</td><td>Washer</td><td>Intake</td></tr>
+            <tr><td>Cleaning</td><td>Washer</td><td>Washing</td></tr>
+            <tr><td>Cleaning</td><td>Washer</td><td>Drain</td></tr>
```

Comment:
`Cleaning` itself isn't a Parallel State, but `Cleaning` is part of the `Washer` parallel region. Modify the parallel state table to reflect this:

```suggestion
            <tr><td>DishwasherOn</td><td>Washer</td><td>Intake</td></tr>
            <tr><td>DishwasherOn</td><td>Washer</td><td>Washing</td></tr>
            <tr><td>DishwasherOn</td><td>Washer</td><td>Drain</td></tr>
```